src/test/javascript/spec/app/entities/conflict/conflict-reducer.spec.ts

```diff
@@ -122,6 +123,7 @@ describe('Entities reducer tests', () => {
 
   describe('Successes', () => {
     it('should fetch all entities', () => {
+      const headers = { ['x-total-count']: 42 };
```

Comment:
Why 'x-total-count' is in array?

Reply:
It was an array, because I copied it from another file. After deleting an array, it still works fine.